features/facebookImport/test/ministory/advertising-value-analysis.test.js

```diff
@@ -0,0 +1,97 @@
+import AdvertisingValueAnalysis from "../../src/model/analyses/ministories/advertising-value-analysis";
+import FacebookAccount from "../../src/model/entities/facebook-account";
+import { runAnalysisForAccount } from "../utils/analyses-execution";
+import {
+    expectInactiveAnalysis,
+    expectAnalysisSuccessStatus,
+    expectActiveAnalysis,
+} from "../utils/analysis-assertions";
+
+describe("Your advertising value analysis from account with no ad interests", () => {
+    let analysis = null;
+    let status = null;
+
+    beforeAll(async () => {
+        const facebookAccount = new FacebookAccount();
+        ({ analysis, status } = await runAnalysisForAccount(
+            AdvertisingValueAnalysis,
+            facebookAccount
+        ));
+    });
+
+    it("has success status", async () => {
+        expectAnalysisSuccessStatus(status);
+    });
+
+    it("is not active", async () => {
+        expectInactiveAnalysis(analysis);
+    });
+});
+
+const DATASETS = [
+    {
+        adInterests: ["Interest"],
+        numberInterests: 1,
+        randomAdInterestsCount: 1,
+    },
+    {
+        adInterests: ["InterestOne", "InterestTwo"],
+        numberInterests: 2,
```

Comment:
Sorry, is this simply the length of the array above?

Reply:
Actually yes. To be removed.